### PR TITLE
Chore/create pull_request_template.md file

### PR DESCRIPTION
**Pull request by tynandebold**
We don't have a PR template file and probably should. This adds one.

### CODE REVIEW

**Review by studioswong — APPROVED** [review]
LGTM

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+<!-- Why was this PR created? -->
+
+## Development notes
+
+<!-- What have you changed? Consider adding a screenshot or GIF. -->
+
+## QA notes
+
+<!-- How has the expected behaviour changed? What testing strategies have you used? -->
```
Comment (studioswong) [inline .github/pull_request_template.md:11]: Might be worth adding a mention to involve designers in QA -
```suggestion
<!-- How has the expected behaviour changed? What testing strategies have you used? Have you QAed your final work with the design team? -->
```

**Review by studioswong — APPROVED** [review]
LGTM, see minor comment below

File: .github/pull_request_template.md (hunk 1 above)
Comment (tynandebold) [reply .github/pull_request_template.md:11]: Nice. Will add.

**Pull request merged**